templates/test/comments.html.ep

```diff
@@ -0,0 +1,37 @@
+% if (!$job->comments->all) {
+    <i>No comments</i>
+% } else {
+    % for my $comment ($job->comments) {
+        <a class="pull-left" href="#">
+            <img class="media-object img-circle" src="<%= $comment->user->gravatar(60) %>" alt="profile">
+        </a>
+        <div class="media-body comment-body">
+            <div class="well well-lg">
+                <h4 class="media-heading"><%= $comment->user->name %> wrote <abbr class="timeago" title="<%= $comment->t_created->datetime() %>Z"><%= format_time($comment->t_created) %></abbr></h4>
+                <p class="media-comment">
+                    %= $comment->rendered_markdown
+                </p>
+            </div>
+        </div>
+    % }
+% }
+
+% if (current_user) {
+    %= form_for 'add_comment', method => "post", class => "form-horizontal", id => "commentForm", role => "form" => begin
+        <div class="form-group">
```

Comment:
Is is intentional that the comment input box is moved a little to the right (with respect to displayed comments)? Because it feels like I'm replying to other comments even when adding unrelated (and knowing we don't have threading).